ipatests/prci_definitions/gating.yaml

```diff
@@ -7,10 +7,10 @@ topologies:
     name: master_1repl
     cpu: 4
     memory: 6450
-  master_1repl_1client: &master_1repl_1client
-    name: master_1repl_1client
+  master_2repl_1client: &master_2repl_1client
```

Comment:
Please use the symlink and ipatests/prci_definitions/temp_commit.yaml instead

Reply:
The symlink should be at the root of the repository btw.

Reply:
The correct way to use the temp commit is the following:
- don't modify gating.yaml
- edit ipatests/prci_definition/temp_commit.yaml and add the test case you want to run
- from the root of your repository, run `ln -sf ipatests/prci_definitions/temp_commit.yaml .freeipa-pr-ci.yaml`
- `git add ipatests/prci_definitions/temp_commit.yaml  .freeipa-pr-ci.yaml`
- `git commit`
With these steps, .freeip-pr-ci.yaml will point to the temp_commit.yaml file. PRCI is extracting the list of tests from .freeipa-pr-ci.yaml, and will execute only the ones specified in temp_commit.yaml.